src/clients/circleci/index.ts

```diff
@@ -19,6 +19,16 @@ export const getBaseURL = (useAPISubdomain = false) => {
   return baseURL;
 };
 
+export const getAppURL = (useAppSubdomain = false) => {
+  let baseURL = process.env.CIRCLECI_BASE_URL || 'https://circleci.com';
+
+  if (useAppSubdomain) {
+    baseURL = baseURL.replace('https://', 'https://app.');
+  }
+
+  return baseURL;
+};
```

Comment:
We can remove the arg here. Based on the function name the caller can expects to get back the app url 
```suggestion
export const getAppURL = () => {
  const baseURL = process.env.CIRCLECI_BASE_URL || 'https://circleci.com';
  return baseURL.replace('https://', 'https://app.');
};
```

Reply:
@niubrandon what did you mean by `getAppURL` here? did it mean get the url with `app.` in it or just in general get me the current circleci url?

Reply:
@stiyyagura0901 `getAppURL` is explicitly for the `app.` url.

Instead of the `getBaseURL` function handling both `https://app.circleci.com` & `https://api.circleci.com`, each case has a dedicated function.

Reply:
Addressed in follow up commit [here](https://github.com/CircleCI-Public/mcp-server-circleci/pull/66/commits/28e76aeade896f123b3b42b007a3652a651cef16).